dashboard/modules/reporter/reporter_agent.py

```diff
@@ -300,10 +316,40 @@ def _get_all_stats(self):
         }
 
     @staticmethod
-    def _record_stats(stats):
+    def _record_stats(stats, cluster_stats):
         records_reported = []
-
         ip = stats["ip"]
+        redis_address = stats["redis_address"]
```

Comment:
if we are using this then what's the use of `self._redis_address`?

Reply:
we don't need this now because of `self._is_head_node`